shared/message/vision/VisualMesh.proto

```diff
@@ -24,18 +24,24 @@ package message.vision;
 import "google/protobuf/timestamp.proto";
 import "Matrix.proto";
 
+/**
+ * Contains the details of a Visual Mesh™ measurement
+ */
 message VisualMesh {
     message Row {
         float phi      = 1;
         int32 segments = 2;
     }
-    uint32                    camera_id       = 1;
-    fmat                      rays            = 2;
-    imat                      neighbourhood   = 3;
-    fmat                      classifications = 4;
-    fmat                      coordinates     = 5;
-    repeated int32            indices         = 6 [packed = true];
-    mat4                      Hcw             = 7;
-    repeated Row              mesh            = 8;
-    google.protobuf.Timestamp timestamp       = 9;
+    /// The camera which took the measurements
+    uint32 camera_id = 1;
+    /// Unit vectors with the origin at the camera position, which represent rays of light
+    fmat           rays            = 2;
+    imat           neighbourhood   = 3;
+    fmat           classifications = 4;
```

Comment:
For a given ray, this provides you the confidence level that this ray belongs to each of the classes (how confident the network is that this ray points to a ball, or a goal post, or a field line, etc). The ordering of the columns is defined when the network is trained. The shape of this matrix is `N x M`, where `N` is the number of classes and `M` is the ray index.